### PR TITLE
{devel}[GCCcore/7.3.0] libsigc++ v2.10.1 (GTK+3)

### PR DESCRIPTION
**Pull request by jose-d**
`libsigc++` is dependency of `GTK+3`.

(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jose-d** [comment]
Test report by @jose-d
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
koios2 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 2.7.5
See https://gist.github.com/97b22b39fa8b81bde9b41b5ea752e6e2 for a full test report.

File: easybuild/easyconfigs/l/libsigc++/libsigc++-2.10.1-GCCcore-7.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['c9a25f26178c6cbb147f9904d8c533b5a5c5111a41ac2eb781eb734eea446003']
+
+builddependencies = [
+    ('binutils', '2.30'),
```
Comment (boegel) [inline easybuild/easyconfigs/l/libsigc++/libsigc++-2.10.1-GCCcore-7.3.0.eb]: @jose-d The existing easyconfig also include `Autotools` as a build dependency, which is required (for `M4`), see also #5787, please include it here too.

**Comment by jose-d** [comment]
Test report by @jose-d
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
koios1 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 2.7.5
See https://gist.github.com/1b2c2f877fea7c081c75a89b883c0d09 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3171.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/aa69a8550e5a176bc62758d1d2bf3744 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2594.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/32f384841314a57fd9ee2b7caa7127bf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2073.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9c9064369a2c016c4bf2a9e61895713f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3171.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/0cadb7db82bb30be180b943ff4380851 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @jose-d!

**Pull request merged**